cfgmgr/vrfmgr.cpp

```diff
@@ -137,6 +140,41 @@ bool VrfMgr::setLink(const string& vrfName)
     return true;
 }
 
+void VrfMgr::handleVnetConfigSet(KeyOpFieldsValuesTuple &t)
+{
+    SWSS_LOG_ENTER();
+
+    string tunnelName;
+    vector<FieldValueTuple> tunnelValues;
+    vector<FieldValueTuple> vnetValues;
+
+    const auto& values = kfvFieldsValues(t);
+    const auto& vrfName = kfvKey(t);
+
+    for (const auto& fv : values)
+    {
+        const auto& name = fvField(fv);
+
+        if (name == "vni")
+        {
+            tunnelValues.push_back(fv);
+        }
+        else if (name == "tunnel")
```

Comment:
"vxlan_tunnel" - This is the name in schema.

Reply:
Yes, I was looking at a diagram, thanks
Done